conbench/app/benchmarks.py

```diff
@@ -11,6 +11,12 @@
 from ..config import Config
 
 
+class UpdateForm(flask_wtf.FlaskForm):
+    is_step_change = w.BooleanField(
+        "Is this result a step change?", render_kw={"onchange": "this.form.submit()"}
+    )
```

Comment:
You might get a second notification about this, I thought I sent this already, but I can't find it on github (it might have been eaten by browser cache?). Aaanyway:

This is relatively minor, and I'm not sure if it's a flask thing or a bootstrap thing (or something else): Is there anyway we could make this a toggle button instead of a checkbox? I fear that checkboxes look more like todo lists (e.g. something that should generally be set) whereas for this it should be relatively rare that it's toggled.

Reply:
oh sure let me mess around with it!

Reply:
Changed to a button; see the screenshots I just posted

Reply:
Would it be possible to have some styling on that button to make it clear when it's marked and when it's not? This also might already exist and I'm only seeing screenshots of un-set buttons so far?

Reply:
How's this? The possible button colors are [here](https://getbootstrap.com/docs/3.3/components/#btn-dropdowns-single)
<img width="370" alt="Screenshot 2023-01-12 at 14 19 37" src="https://user-images.githubusercontent.com/16600275/212172489-59700850-1842-463d-8697-91f10766be0a.png">
<img width="636" alt="Screenshot 2023-01-12 at 14 19 46" src="https://user-images.githubusercontent.com/16600275/212172503-f135508b-78cd-4d7b-a774-c77beca3bcdd.png">